conans/client/generators/cmake_find_package.py

```diff
@@ -3,19 +3,25 @@
 
 
 generic_find_package_template = """
+include(FindPackageHandleStandardArgs)
+
 message(STATUS "Conan: Using autogenerated Find{name}.cmake")
 # Global approach
 SET({name}_FOUND 1)
+SET({name}_VERSION "{version}")
 SET({name}_INCLUDE_DIRS {deps.include_paths})
 SET({name}_INCLUDES {deps.include_paths})
 SET({name}_DEFINITIONS {deps.defines})
 SET({name}_LIBRARIES "") # Will be filled later
 SET({name}_LIBRARIES_TARGETS "") # Will be filled later, if CMake 3
 SET({name}_LIBS "") # Same as {name}_LIBRARIES
 
-mark_as_advanced({name}_FOUND {name}_INCLUDE_DIRS {name}_INCLUDES
-                 {name}_DEFINITIONS {name}_LIBRARIES {name}_LIBS)
+find_package_handle_standard_args({name}
```

Comment:
Could this (https://cmake.org/cmake/help/v3.0/module/FindPackageHandleStandardArgs.html) additionally help with the casing of the package names when using this generator or nothing to do?

Reply:
The thing is that more recent versions of cmake mark that FOUND_VARIABLE parameter as obsolete.  See for example https://cmake.org/cmake/help/v3.13/module/FindPackageHandleStandardArgs.html which states "This exists only for compatibility with older versions of CMake and is now ignored. Result variables of both names are always set for compatibility."